README.md

```diff
@@ -96,6 +96,7 @@ Session store-related settings:
 | - | - | - |
 | `SESSION_STORE_PATH` | "/var/lib/authservice/data.db" | Path to local session store. Backed by BoltDB. |
 | `SESSION_MAX_AGE` | "86400" | Time in seconds after which sessions expire. Defaults to a day (24h). |
+| `COOKIE_SAME_SITE` | "Strict" | Allow you to specify the SameSite attribute of the Set-Cookie HTTP response header. Check details of SameSite attribute [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite). Its value could be "None", "Lax" or "Strict"  |
```

Comment:
Can you make a small clarification:

name: `SESSION_SAME_SITE`

default: "Lax"

```
SameSite attribute of the session cookie. The [SameSite](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite) attribute acts as an extra protection against CSRF. Its value can be "None", "Lax" or "Strict".
```

Reply:
done